me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java

```diff
@@ -8,53 +8,50 @@
 import android.webkit.CookieManager;
 import android.webkit.WebView;
 
-
-public class WebViewActivity extends ActionBarActivity
-{
-    IDmeWebVerify iDmeWebVerify;
-    private WebView webView;
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState)
-    {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_web_view);
-
-
-        getSupportActionBar().setTitle(Html.fromHtml("<font color='#2fc073'>Verify With ID.me </font>"));
-        getSupportActionBar().setBackgroundDrawable(new ColorDrawable(0xff2e3d50));
-        String scope = getIntent().getStringExtra("scope");
-        String url = getIntent().getStringExtra("URL");
-        String clientId = getIntent().getStringExtra("clientID");
-        String redirectUri = getIntent().getStringExtra("redirectURI");
-        boolean returnProperties = getIntent().getBooleanExtra("returnProperties", true);
-
-        iDmeWebVerify = new IDmeWebVerify(clientId, redirectUri, scope, this, returnProperties);
-
-        webView = (WebView) findViewById(R.id.webView);
-        webView.setWebViewClient(iDmeWebVerify.getWebViewClient());
-        webView.loadUrl(url);
-
-    }
-
-    @Override
-    public void onDestroy()
-    {
-        super.onDestroy();
-        CookieManager cookieManager = CookieManager.getInstance();
-
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP)
-        {
-            cookieManager.removeAllCookie();
-        }
-        else
-        {
-            cookieManager.removeAllCookies(null);
-        }
-        webView.clearCache(true);
-        webView.clearHistory();
-        webView.clearFormData();
-        webView.destroy();
-
+public class WebViewActivity extends ActionBarActivity {
```

Comment:
is necessary that this class be public? maybe it can package protected. If not, I would change its name. It is much generic and it could leak in confusion by the SDK users. As it is enclosed in the SDK package this wouldn't be a big deal, but I would consider renaming it